front/app/containers/App/index.tsx

```diff
@@ -340,6 +332,21 @@ const App = ({ children }: Props) => {
   const { pathname } = removeLocale(location.pathname);
   const showFrontOfficeNavbar = !isAdminPage || isPagesAndMenuPage;
 
+  // Ensure authUser is loaded before rendering the app
+  if (!authUser && isLoading) {
```

Comment:
Do we need the `!authUser` check here? Or would just `isLoading` be fine? :thinking: